articles/azure/azs-how-create-S2S-VPN-azurestack-to-azure.md

```diff
@@ -92,10 +94,16 @@ To associate a virtual network with a gateway, it must first contain a valid gat
 
 2. In the **Settings** section of the virtual network blade, select **Subnets**.
 
+    ![Subnets button in virtual networks blade](images/azs-public-browser-virtual-network-subnets-button.png)
+
 3. On the **Subnets** page, click the **Gateway Subnet** button.
 
+    ![Gateway subnet button in virtual networks blade](images/azs-public-browser-virtual-network-gateway-subnet-button.png)
+
 4. Fill in the **Address range** and add a route table if required, then click **OK**. The subnet should now have appeared.
 
+    ![Create gateway subnet](images/azs-browser-create-gateway-subnet.png)
```

Comment:
Image isn't showing. Maybe the case of png?

Reply:
Resolved by fixing file extension.